src/repeat.test.js

```diff
@@ -20,46 +20,50 @@ const testCases = [
   { string: 'abc', count: 2, toBe: 'abcabc' },
   { string: 'abc', count: 3, toBe: 'abcabcabc' },
   { string: 'abc', count: 4, toBe: 'abcabcabcabc' },
-];
+]
 
 describe('repeat (when String.repeat exist)', () => {
   testCases.forEach(testCase => {
-    test(`should return ${testCase.toBe} on string ${testCase.string} with count ${testCase.count}`, () => {
-      expect(repeat(testCase.string, testCase.count)).toBe(testCase.toBe);
-    });
-  });
-});
+    test(`should return ${testCase.toBe} on string ${
+      testCase.string
+    } with count ${testCase.count}`, () => {
+      expect(repeat(testCase.string, testCase.count)).toBe(testCase.toBe)
+    })
+  })
+})
 
 describe('repeat (when String.repeat does not exist)', () => {
   testCases.forEach(testCase => {
-    test(`should return ${testCase.toBe} on string ${testCase.string} with count ${testCase.count}`, () => {
+    test(`should return ${testCase.toBe} on string ${
+      testCase.string
+    } with count ${testCase.count}`, () => {
       expect(
-        repeat(createStringWithoutRepeat(testCase.string), testCase.count),
-      ).toBe(testCase.toBe);
-    });
-  });
+        repeat(createStringWithoutRepeat(testCase.string), testCase.count)
+      ).toBe(testCase.toBe)
+    })
+  })
 
   test('should throw a range error when count is negative', () => {
     expect(() => {
-      repeat(createStringWithoutRepeat('abc'), -1);
-    }).toThrowErrorMatchingSnapshot();
-  });
+      repeat(createStringWithoutRepeat('abc'), -1)
+    }).toThrowErrorMatchingSnapshot()
+  })
 
   test('should throw a range error when count is negative infinity', () => {
```

Comment:
Nitpick: Could you convert all the `test(` to `it(` so that it is consistent with the rest of our projects?